packages/@lwc/compiler/src/options.ts

```diff
@@ -128,7 +129,7 @@ export interface TransformOptions {
     instrumentation?: InstrumentationObject;
     /** API version to associate with the compiled module. Values correspond to Salesforce platform releases. */
     apiVersion?: number;
-    targetSSR?: boolean;
+    targetSSR?: CompilationMode | null;
```

Comment:
Changing this boolean would be awkward because there is already a `targetSSR` boolean that is plumbed throughout much of core/`lwc-platform-public`. I'd suggest adding a new `targetSSRMode` option.

Reply:
✅